src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/CaptureCollection.cs

```diff
@@ -109,7 +109,7 @@ public void CopyTo(Capture[] array, int arrayIndex)
 
         int IList<Capture>.IndexOf(Capture item)
         {
-            var comparer = EqualityComparer<Capture>.Default;
+            EqualityComparer<Capture> comparer = EqualityComparer<Capture>.Default;
```

Comment:
Separately, I would suggest looking at not storing this into a local first.  With devirtualization support for EqualityComparer.Default, this might be better as:
```C#
for (int i = 0; i < Count; i++)
{
    if (EqualityComparer<Capture>.Default.Equals(this[i], item)
    {
        return i;
    }
}
return -1;
```
There's also a lot of code behind the seemingly innocent `this[i]`; something else to look at separately.